docs/shield-api.md

```diff
@@ -596,6 +835,219 @@ $bunnyHttpClient->request(
 );
 ```
 
+### Access Lists
+
+#### [List Shield Zone Access Lists](https://docs.bunny.net/reference/get_shield-shield-zone-shieldzoneid-access-lists)
+
+```php
+$bunnyHttpClient->request(
+    new \ToshY\BunnyNet\Model\Api\Shield\AccessLists\ListShieldZoneAccessLists(
+        shieldZoneId: 1,
+    )
+);
+```
+
+#### [Get Shield Zone Access List](https://docs.bunny.net/reference/get_shield-shield-zone-shieldzoneid-access-lists-id)
+
+```php
+$bunnyHttpClient->request(
+    new \ToshY\BunnyNet\Model\Api\Shield\AccessLists\GetShieldZoneAccessList(
+        id: 2,
+        shieldZoneId: 1,
+    )
+);
+```
+
+#### [Get Shield Zone Access List Enums](https://docs.bunny.net/reference/get_shield-shield-zone-shieldzoneid-access-lists-enums)
+
+```php
+$bunnyHttpClient->request(
+    new \ToshY\BunnyNet\Model\Api\Shield\AccessLists\GetShieldZoneAccessListEnums(
+        shieldZoneId: 1,
+    )
+);
+```
+
+#### [Create Shield Zone Access List](https://docs.bunny.net/reference/get_shield-shield-zone-shieldzoneid-access-lists-enums)
+
+```php
+$bunnyHttpClient->request(
+    new \ToshY\BunnyNet\Model\Api\Shield\AccessLists\CreateShieldZoneAccessList(
+        shieldZoneId: 1,
+        body: [
+            'name' => 'Custom Access List',
+            'description' => 'This is a description for the access list',
+            'type' => 0,
+            'content' => '192.168.0.1',
+            'checksum' => '37d7a80604871e579850a658c7add2ae7557d0c6abcc9b31ecddc4424207eba3',
+        ],
+    )
+);
+```
+
+??? note
```

Comment:
Add warning note that when you have an existing custom access lists, this will return 202 with the following errorKey `limit_reached.access_list`:

```
Custom Access List limit exceeded. You can have a maximum of 1 custom lists. Please upgrade your Bunny Shield tier or contact support for assistance.
```